src/NuGet.Core/NuGet.Packaging/PackageExtraction/PackageExtractionContext.cs

```diff
@@ -18,5 +20,10 @@ public class PackageExtractionContext
         public PackageSaveMode PackageSaveMode { get; set; } = PackageSaveMode.Defaultv2;
 
         public XmlDocFileSaveMode XmlDocFileSaveMode { get; set; } = PackageExtractionBehavior.XmlDocFileSaveMode;
+
+        public PackageExtractionContext(ILogger logger)
+        {
+            Logger = logger ?? NullLogger.Instance;
```

Comment:
Done

---
In reply to: [60082588](https://github.com/NuGet/NuGet.Client/pull/504#discussion_r60082588) [](ancestors = 60082588)